securitycenter/assets/list_all_project_assets.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Contains examples calls to Cloud Security Center ListAssets API method.
+package assets
+
+// [START list_project_assets]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	securitycenter "cloud.google.com/go/securitycenter/apiv1"
+	"google.golang.org/api/iterator"
+	securitycenterpb "google.golang.org/genproto/googleapis/cloud/securitycenter/v1"
+)
+
+// listAllProjectAssets lists all current GCP project assets in orgID and
+// prints out results to w. listAllProjectAssets returns the number of project
+// assets found.  orgID is the numeric organization ID of interest.
+func listAllProjectAssets(w io.Writer, orgID string) (int, error) {
+	// orgID := "12321311"
+	// Instantiate a context and a security service client to make API calls.
+	ctx := context.Background()
+	client, err := securitycenter.NewClient(ctx)
+	if err != nil {
+		return -1, fmt.Errorf("Error instantiating client %v\n", err)
+	}
+	defer client.Close() // Closing the client safely cleans up background resources.
+	onlyProjects := "security_center_properties.resource_type=" +
```

Comment:
No need for a newline. Can use \` to avoid having to escape the `"`.

It would probably be OK to inline this when creating `req`. It's common to nest declarations of Go protos - fewer variables and less indirection.

```suggestion
	onlyProjects := `security_center_properties.resource_type="google.cloud.resourcemanager.Project"`
```

Reply:
inlined.

Reply:
done.